File: zig/arr.cpp
#include <iostream>





int main() {

    int arr[100000] = {0};
    int len = 0;

    for(int i = 0; i < 100000; i++) {
        arr[i] = 2 * 5 / 1 + 11 + 33 - 99;
        len++;
    }

    std::cout << "Array len is: " << len << '\n';


    return 0;
}

File: zig/arr.zig
const std = @import("std");
const print = std.debug.print;
const output = std.io.getStdOut();

pub fn main() !void {
    var arr: [100000]i32 = undefined;
    std.debug.print("Set hello {d}\n", .{1});

    for (0..100000) |i| {
        arr[i] = 2 * 5 / 1 + 11 + 33 - 99;
    }
    print("array size is: {}\n", .{arr.len});
}
